cli-docker/README.adoc

```diff
@@ -0,0 +1,100 @@
+= Example Docker image for Decodable CLI
```

Comment:
```suggestion
= Example container image for Decodable CLI
```

My Red Hat past speaking here: a better and more correct term is "container image". Docker is just but one container runtime (other ones being e.g. containerd or podman). An OCI-compliant container image (as e.g. created with Docker and a Dockerfile) can be run using any of these runtimes.

Reply:
LOL, can't help but link to this tweet which i saw earlier this week https://x.com/endingwithali/status/1832882721733095836 🤣

but point taken, I'll commit the suggestion :)

Reply:
> My Red Hat past speaking here

you also missed the opportunity to say "Putting my Red Hat hat on" X-D

Reply:
ROFL.